client/src/intro-exit/VideoCheck.jsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { VideoCall2 } from "../components/VideoCall2";
+import { Button2 } from "../components/Button2";
+
+export default function VideoCheck({next}) {
+
+    const screenStyle = {
+        display: 'flex',
+        flexDirection: 'column',
+        width: '1000px',
+        height: '1000px',
+        margin: '10px',
+        alignItems: 'flex-start',
+        gap: '10px',
+        marginLeft: '50px'
+    }
+
+    const headerStyle = {
+        marginLeft: '10px'
+    }
+
+    const questionsStyle = {
+        display: 'flex',
+        flexDirection: 'column',
+        width: '1000px',
+        height: '500px',
+        margin: '10px',
+        alignItems: 'flex-start',
+        gap: '8px'
+    }
+
+    const [canSee, setChecked1] = React.useState(false);
+    const [noName, setChecked2] = React.useState(false);
+    const [backgroundInfo, setChecked3] = React.useState(false);
+    const [safePlace, setChecked4] = React.useState(false);
+    const [noInterrupt, setChecked5] = React.useState(false);
+    const [speakFree, setChecked6] = React.useState(false);
+    const [enabled, setChecked7] = React.useState(false);
+
+    const Checkbox = ({ label, value, onChange }) => {
+        return (
+          <label>
+            <input type="checkbox" checked={value} onChange={onChange} />
+            {label}
+          </label>
+        );
+      };
+
+    function handleButtonClick() {
+        if (canSee && noName && backgroundInfo && safePlace && noInterrupt && speakFree) {
+            next()
+     }
+    };
+     
+    function handleChange1() {
```

Comment:
You should be able to do this all with one handler. The checkbox can pass its name or value to the handler - see this pattern in the ExitSurvey:
```jsx
function handleEducationChange(e) {
    setEducation(e.target.value);
  }
```